tests/exploreServices.spec.ts

```diff
@@ -251,9 +251,7 @@ test.describe('explore services page', () => {
       await showLogsHeaderBtn.click();
 
       // assert we navigated
-      await expect(
```

Comment:
This will always be true

![imagen](https://github.com/user-attachments/assets/b169177d-0fa3-4ee0-9d94-038f36f12167)

Reply:
Reverted